backend/cognito/cognito.template.yml

```
@@ -90,6 +125,34 @@ Resources:
             Trigger: CustomMessage
             UserPool: !Ref UserPool
 
+  SecurityCodeTextMessageSender:
+    # checkov:skip=CKV_AWS_115: "Ensure that AWS Lambda function is configured for function-level concurrent execution limit"
+    # checkov:skip=CKV_AWS_117: "Ensure that AWS Lambda function is configured inside a VPC"
+    # checkov:skip=CKV_AWS_116: "Ensure that AWS Lambda function is configured for a Dead Letter Queue(DLQ)"
+    # checkov:skip=CKV_AWS_173: "Check encryption settings for Lambda environmental variable"
+    Type: AWS::Serverless::Function
+    Metadata: # Manage esbuild properties
+      BuildMethod: esbuild
+      BuildProperties:
+        Minify: true
+        Target: "es2020"
+        Sourcemap: true
+        EntryPoints:
+          - security-code-text-message-sender.ts
```

```suggestion
    Metadata:
      BuildMethod: esbuild
      BuildProperties:
        Sourcemap: true
```
Copy from other functions

---

addressed